src/helpers/jsonStringify.ts

```diff
@@ -1,12 +1,14 @@
 
+// eslint-disable-next-line header/header
 import * as stableStringify from 'json-stable-stringify';
```

Comment:
I was running into some module + commonjs issues with this since it stableStringify was being set to `{default: function}` instead of the function. I swapped over to `import stableStringify = require('json-stable-stringify');` to get it working but not sure what fix we would want here